modules/videoio/src/cap_v4l.cpp

```diff
@@ -406,10 +407,10 @@ bool CvCaptureCAM_V4L::try_palette_v4l2()
     form.fmt.pix.field       = V4L2_FIELD_ANY;
     form.fmt.pix.width       = width;
     form.fmt.pix.height      = height;
-
-    if (!tryIoctl(VIDIOC_S_FMT, &form))
+    if (!tryIoctl(VIDIOC_S_FMT, &form, 2))
```

Comment:
What is about adding parameter to handle "EBUSY" case?

Reply:
Ok, I will add it as additional argument, before attempts count.
I would like to keep limited attempts to ensure that loop is finite.